src/github.com/getlantern/flashlight/autoupdate/autoupdate.go

```diff
@@ -25,27 +24,28 @@ var (
 	updateMutex sync.Mutex
 
 	httpClient *http.Client
-	watching   int32 = 0
+	watching   int32
 
 	applyNextAttemptTime = time.Hour * 2
 )
 
-func Configure(cfg *config.Config) {
+// Configure sets the CA certificate to pin for the TLS auto-update connection.
+func Configure(updateURL, updateCA string) {
 	cfgMutex.Lock()
 
-	if cfg.UpdateServerURL != "" {
-		updateServerURL = cfg.UpdateServerURL
+	if updateURL != "" {
+		updateServerURL = updateURL
 	}
 
 	go func() {
```

Comment:
The goroutine is not required as `enableAutoupdate` doesn't block.
`cfgMutex` is only used here, can be removed too.